blocks/rapid-result/rapid-result.js

```diff
@@ -0,0 +1,139 @@
+import { readBlockConfig, fetchPlaceholders, decorateIcons } from '../../scripts/aem.js';
+import { createElement, observe } from '../../scripts/blocks-utils.js';
+
+// TODO: This is dummy function that fetch sample data from EDS json.
+// It will be replaced when API call is available.
+async function fetchRapidResultMockData() {
```

Comment:
Perhaps we could put this in a scripts/mockapi.js to clean them all up at once.

Reply:
+1

Reply:
Okay, I'll move it to mockapi.js. (It was there in the initial version, but one day I found that js got changed a lot, to avoid accident deleting I kept it in the block)